mopidy_pandora/uri.py

```diff
@@ -38,6 +40,7 @@ def __repr__(self):
     def encoded_attributes(self):
         encoded_dict = {}
         for k, v in list(self.__dict__.items()):
+            print(f"Ecoding {v}\n")
```

Comment:
Please log instead.

Reply:
Once again this was temporary, I'll remove.